client/state/reader/feed-searches/test/reducer.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import freeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { receiveFeedSearch, } from '../actions';
```

Comment:
Ha. Not sure we have a Calypso standard on this, but we mostly use `only-multiline` elsewhere in the codebase:

http://eslint.org/docs/rules/comma-dangle